cfme/infrastructure/provider/__init__.py

```diff
@@ -271,10 +225,12 @@ def resetter(self):
 
 @navigator.register(InfraProvider, 'Add')
 class Add(CFMENavigateStep):
+    VIEW = ProviderAddView
     prerequisite = NavigateToSibling('All')
 
     def step(self):
-        cfg_btn('Add a New Infrastructure Provider')
+        cfg = self.prerequisite_view.toolbar.configuration
```

Comment:
dropdown is one line below. full call just didn't fit one line so I had to split it up.